app/components/GuideReviewForm/index.js

```diff
@@ -0,0 +1,116 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import styled from 'styled-components';
+import Select from 'components/Select';
+
+import PageSection from 'components/PageSection';
+import BorderBottomDiv from 'components/shared/BorderBottomDiv';
+import Stars from 'components/Stars';
+
+class GuideReviewForm extends React.Component {
```

Comment:
Can you think of another name for this? It's not really a guide review form, it's a select-guide and rate-guide section of  a trip report form. No problem if not.

Also, should we put this file in the WriteReviewPage folder? Do you expect we'll reuse this component anywhere else? it seems tightly coupled to the other stuff there, e.g. https://github.com/Octember/the-approach/pull/39/files#diff-0c7f0a8dbc9577f25b87a4c21c69b23eR21

I think this belongs in the folder with the other WriteReviewForm stuff

Reply:
We could call it a card, I suppose. (I was noticing Bootstrap added cards to the latest version; could be useful later on if we wanted to integrate it into the app.)

I don't have the overall vision of the project, so I was thinking...maybe it could be used elsewhere? If we add it to the WriteReviewForm folder, will it still be an index.js in its own folder or something like GuideReviewCard.js?

Reply:
We can always split it out later, for now I think it's coupled. `GuideReviewCard.js` sounds good to me!